test_scripts/API/VehicleData/commonVehicleData.lua

```diff
@@ -359,31 +360,72 @@ function m.updatePreloadedFile(pUpdateFunc)
   m.setPreloadedPT(pt)
 end
 
-function m.processGetVDsuccess(pData)
+function m.processGetVDsuccess(pData, pHmiResParams)
+  pHmiResParams = pHmiResParams or m.allVehicleData[pData]
+
+  local hmiResParams = {
+    [pData] = pHmiResParams.value
+  }
   local reqParams = {
-     [pData] = true
+    [pData] = true
+  }
+
+  m.getVehicleDataSuccessExpectation(reqParams, reqParams, hmiResParams)
+end
+
+function m.processGetVDsuccessManyParameters( ... )
+  local reqParams = {}
+  local hmiResParams = {}
+
+  for _, vehicleData in ipairs( { ... } ) do
```

Comment:
@ydementieiev Do we need `ipairs` here?

Reply:
@AByzhynar  in `processGetVDsuccessManyParameters` we pass many arguments, and with ipairs( { ... } ) have access to all of them